nix/deku-p/service.nix

```diff
@@ -7,7 +7,7 @@
 with lib; let
   cfg = config.services.deku-node;
   listToString = lib.strings.concatStringsSep ",";
-  environment = builtins.mapAttrs (_: value: builtins.toString value) cfg.environment;
+  cookieVM = "${pkgs.nodejs-16_x}/bin/node ${deku-packages.${config.nixpkgs.system}.decookies-vm}/lib/node_modules/decookies-vm/lib/src/index.js"
```

Comment:
All the changes to this file should be discarded.

Reply:
done